prow/plugins/config.go

```diff
@@ -1518,8 +1546,27 @@ func ResolveBugzillaOptions(parent, child BugzillaBranchOptions) BugzillaBranchO
 		}
 		output.DependentBugStates = mergeStatusesIntoStates(output.DependentBugStates, child.DependentBugStatuses)
 	}
-	if child.DependentBugTargetRelease != nil {
-		output.DependentBugTargetRelease = child.DependentBugTargetRelease
+	if child.DependentBugTargetReleases != nil {
+		output.DependentBugTargetReleases = child.DependentBugTargetReleases
+	}
+	if child.DeprecatedDependentBugTargetRelease != nil {
+		warnDeprecated(&warnDependentBugTargetRelease, 5*time.Minute, "Please update plugins.yaml to use dependent_bug_target_releases instead of the deprecated dependent_bug_target_release")
+		contains := false
+		if child.DependentBugTargetReleases != nil {
+			for _, target := range *child.DependentBugTargetReleases {
+				if target == *child.DeprecatedDependentBugTargetRelease {
+					contains = true
+					break
+				}
+			}
+		}
+		if !contains {
+			if output.DependentBugTargetReleases == nil {
+				output.DependentBugTargetReleases = &[]string{*child.DeprecatedDependentBugTargetRelease}
+			} else {
+				_ = append(*output.DependentBugTargetReleases, *child.DeprecatedDependentBugTargetRelease)
```

Comment:
Unless I'm crazy, this dereferences `output.DependentBugTargetRelease` and allocates for a new slice (by copy) that appends the child's release, so it does not change the output. Do we test this?

Reply:
Yeah, looks like this was cutting too many corners.  Stopped assuming `append` mutated in place with 92e9d951dd.